### PR TITLE
schismtracker: Add version 20190805

### PR DESCRIPTION
**Pull request by shytikov**
Manifest supports both `32` and `64` bit versions.

### CODE REVIEW

File: bucket/schismtracker.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/schismtracker/schismtracker/releases/download/$version/schismtracker-$version-win64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/schismtracker/schismtracker/releases/download/$version/schismtracker-$version-win32.zip"
+            }
+        }
+    }
+}
```
Comment (Ash258) [inline bucket/schismtracker.json]:
```suggestion
}

```

File: bucket/schismtracker.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "bin": "schismtracker.exe",
+    "shortcuts": [
+        [
+            "schismtracker.exe",
+            "Schism Tracker"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/schismtracker/schismtracker/"
```
Comment (Ash258) [inline bucket/schismtracker.json]:
```suggestion
        "github": "https://github.com/schismtracker/schismtracker"
```

File: bucket/schismtracker.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+    "homepage": "http://schismtracker.org/",
```
Comment (Ash258) [inline bucket/schismtracker.json]:
```suggestion
    "homepage": "http://schismtracker.org",
```

File: bucket/schismtracker.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "version": "20190805",
+    "description": "An oldschool sample-based music composition tool.",
```
Comment (Ash258) [inline bucket/schismtracker.json]:
```suggestion
    "description": "An oldschool sample-based music composition tool",
```

**Comment by shytikov** [comment]
@Ash258 thank you for all the comments! I have accepted proposed resolution for all of them!

**Pull request merged**